src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs

```diff
@@ -668,7 +693,7 @@ public static bool AddTopLevelSuggestionsForGivenNode(IntellisenseData.Intellise
                 }
             }
 
-            if (callNode.Parent != null)
+            if (callNode.Parent != null && node is not ErrorNode)
```

Comment:
>&& node is not ErrorNode [](http://example.com/codeflow?start=40&length=24)

what kind of expression needs this condition? can you please give an example?

Reply:
Removed